### PR TITLE
Add job names configuration

### PR DESCRIPTION
**Pull request by Jasper-Nelligan**
Add job names configuration which is automatically filled with the file's name, minus the .stl extension. The job name is limited to 100 characters.

Also prepends the stl file in our database with the job name so that the user can know which file they're downloading. 

Screenshot (55)

### CODE REVIEW

File: src/backend.js [hunk 1]
```diff
@@ -114,7 +115,8 @@ export async function uploadThumbnail(thumbnail, id) {
   uploadBytes(storageRef, thumbnailBlob);
 };
 
-export async function uploadStl(stlFile, id) {
-  var storageRef = ref(firebaseStorage, `print-files/${id}.stl`);
-  uploadBytes(storageRef, stlFile);
+export async function uploadStl(file, jobName, id) {
+  console.log(jobName + "_" + id + ".stl")
+  var storageRef = ref(firebaseStorage, `print-files/${jobName + "_" + id}.stl`);
+  uploadBytes(storageRef, file);
```
Comment (mas393) [inline src/backend.js]: Does adding `jobName` to the stored filename provide any benefit? Id is already a unique identifier.

**Review by mas393 — COMMENTED** [review]
Job names working as expected for me

File: src/backend.js (hunk 1 above)
Comment (mas393) [reply src/backend.js]: I removed the jobName from the filename when fixing merge issues. I can add it back in if you think its needed

Comment (Jasper-Nelligan) [reply src/backend.js]: Hi, yes the jobName is there to provide a readable file name for when the user downloads it to their local machine. That way the user knows which file is associated with which job they've accepted

Comment (mas393) [reply src/backend.js]: Okay I'll switch it back then approve the PR

Comment (Jasper-Nelligan) [reply src/backend.js]: Thanks Matt, I appreciate the review

**Review by mas393 — APPROVED** [review]
👍

**Pull request merged**